internal/store/store.go

```diff
@@ -108,20 +110,25 @@ type Store struct {
 // CacheStores stores cache.Store for all Kinds of k8s objects that
 // the Ingress Controller reads.
 type CacheStores struct {
+	// Core Kubernetes Stores
 	IngressV1beta1 cache.Store
 	IngressV1      cache.Store
-	TCPIngress     cache.Store
-	UDPIngress     cache.Store
+	Service        cache.Store
+	Secret         cache.Store
+	Endpoint       cache.Store
 
-	Service  cache.Store
-	Secret   cache.Store
-	Endpoint cache.Store
+	// Gateway APIs Stores
```

Comment:
```suggestion
	// Gateway API Stores
```

Reply:
changed in eda82b7f62d2d7222954ddda8a3276f3cae37a6e